test/tests/carousel-1.js

```diff
@@ -15,6 +15,7 @@ const exampleFile = 'carousel/carousel-1/carousel-1.html';
 
 const ex = {
   landmarkSelector: '#myCarousel',
+  buttonSelector: '#ex1 button',
   previousNextButtonSelector: '#ex1 .carousel-control',
```

Comment:
This class has been removed, so this selector doesn't select anything in the test that uses it!

Reply:
sure you can remove it